cromwellApiClient/src/main/scala/cromwell/api/CromwellClient.scala

```diff
@@ -130,9 +130,18 @@ class CromwellClient(val cromwellUrl: URL,
     simpleRequest[WorkflowOutputs](outputsEndpoint(workflowId, args))
   }
 
-  def labels(workflowId: WorkflowId, headers: List[HttpHeader] = defaultHeaders)
+  def labels(workflowId: WorkflowId,
```

Comment:
If you feel strongly, you could call this `getLabels`. I'm not too worried about that though.